odbc/src/tests.rs

```diff
@@ -463,3 +465,326 @@ mod get_diag_rec {
         );
     }
 }
+mod env_attributes {
```

Comment:
> Can you add tests with ValuePtr being Null for all environment attributes ?

Done.

> should we add tests where ValuePtr is an invalid value

I don't think this case needs to be tested anymore now that I have an enum for every environment attribute. Typecasting an invalid value to an environment attribute enum variant will result in a successful cast instead of an error, so I think just testing the individual variants should be enough.

Reply:
Awesome work, I like this new environment attributes test layout a lot!